### DISCUSSION TITLE
bleak as "server" (advertise services with characteristics)

### DISCUSSION THREAD

**Discussion by alex9099** [hbldh/bleak#1351] [Q&A] [answered]
Hey,
I'm wanting to have my computer act as a server so other devices could connect to it (so it would act like any ble enabled device), is this even possible? Didn't find any example for it, only to work as a client

**Comment by dlech** [comment, marked as answer]
It sounds like you want https://github.com/kevincar/bless/

**Reply by alex9099** [reply]
Thanks! I think that would be what i'm looking for, perhaps a link to bless on the readme could be helpful (like they have for bleak on their readme)

**Reply by dlech** [reply]
Happy to take a pull request to add this.
